### DISCUSSION TITLE
Police Law

### DISCUSSION THREAD

**Discussion by michalptacnik** [liberland/laws#33] [Ideas]
Advisor for Legislature:
- instead of having the police as a State body operating over districts and areas...
- ... we can have private Security Agencies for the local stuff...
- ... and State Police for emergencies only. 
- There should not be too many officers (10 or 20), and they should have 1 or 2 offices, highly equipped. 
- Jobs: secure border, Interpol liasoning, matters of national security.
- For everything else: employ private Security Agencies.
- Have guidelines as to matters like appeal or Congress control.
- Have 1 or 2 Liberland Police Boats in front of the territory.

Advisor for Blockchain:
- This doesn't solve the problem of not having a group that helps liberlanders who can't pay for the Security Agencies;
- One can resolve this by making the public Police sufficiently powerful - make it "future proof";
- While she understands the MOJ remark (see below), the Advisor suggests creating a sunset clause for the State police.

MOJ:
- This is a good system for the future; for the present, we need to have one State police which works in a way predictable by other States.
- Sunset clause = 2000 citizens; after that, an amendment which removes much of the current Law, enabling the vision of the Advisor for Legislature.

**Comment by ghost** [comment]
In addition to private Security Agencies + State Police,
I think we should strongly : 
- encourage and promote self-defence, 
- as well as free associations of citizen with explicit goal of helping with law enforcement (this exists already in several countries).

This would :
- encourage individual responsibility on one hand (+ encourage good knowledge of the law)
- and may help to reduce the costs of private Security Agencies + State Police.

(New-Hampshire maybe a source of inspiration : 
![image](https://github.com/liberland/laws/assets/142343456/3df89da0-0c49-40df-8c40-74b39f9473b8) )
